roundabout/test/piece.test.js

```diff
@@ -0,0 +1,58 @@
+import { test } from './helpers/context.js'
+import { CID } from 'multiformats/cid'
+import * as Raw from 'multiformats/codecs/raw'
+import { sha256 } from 'multiformats/hashes/sha2'
+import { Piece, MIN_PAYLOAD_SIZE } from '@web3-storage/data-segment'
+import { findEquivalentCarCids, asCarCid, asPieceCid, CAR_CODE } from '../piece.js'
+
+test('findEquivalentCarCids', async t => {
+  const bytes = new Uint8Array(MIN_PAYLOAD_SIZE)
+  const pieceCid = Piece.fromPayload(bytes).link
+  const carCid = CID.createV1(CAR_CODE, sha256.digest(bytes))
+  const rawCid = CID.createV1(Raw.code, sha256.digest(bytes))
+  const carSet = await findEquivalentCarCids(pieceCid, async () => {
+    return [
+      { type: 'assert/equals', content: pieceCid, equals: carCid }, // yes! is equivalent carCid
+      { type: 'assert/equals', content: carCid, equals: pieceCid }, // no: is duplicate
+      { type: 'assert/equals', content: pieceCid, equals: rawCid }, // no: pieceCId not mapped to carCid
+      { type: 'assert/location', content: pieceCid, location: `ipfs://${carCid}` }, // no: is not equals claim
+    ]
+  })
+  t.is(carSet.size, 1, 'should return 1 unique carCid')
+  t.is([...carSet].at(0).toString(), carCid.toString())
+})
+
+test('findEquivalentCarCids from content-claims api', async t => {
+  const pieceCid = CID.parse('bafkzcibbai3tdo4zvruj6zxo6wlt4suu3imi6to4vzmaojh4n475mdp5jcbtg')
+  const carCid = CID.parse('bagbaieratdhefxxpkhkae2ovil2tcs7pfr2grvabvvoykful7k2maeepox3q')
```

Comment:
CIDs are the ones generated by the previous test.

Added to the content-claims api manually
```shell
❯ ./bin.js equals bafkzcibbai3tdo4zvruj6zxo6wlt4suu3imi6to4vzmaojh4n475mdp5jcbtg bagbaieratdhefxxpkhkae2ovil2tcs7pfr2grvabvvoykful7k2maeepox3q > equals-demo.claim

❯ ./bin.js write equals-demo.claim 
{ ok: {} }
```

Reply:
Note: the claims added using the CLI will expire and then not be retrievable. I will open an issue for setting it.

Reply:
https://github.com/web3-storage/content-claims/issues/28

Reply:
Fixed! a never expiring claim has now been written

```shell
❯ ./bin.js equals bafkzcibbai3tdo4zvruj6zxo6wlt4suu3imi6to4vzmaojh4n475mdp5jcbtg bagbaieratdhefxxpkhkae2ovil2tcs7pfr2grvabvvoykful7k2maeepox3q -e never -o equals-demo-2.claim
bafyreibb5x5sw2kw6kcr7pjrho5evli7e3n4uohc63ek3vh5dxvw42feeq

❯ ./bin.js inspect equals-demo-2.claim 
{
  iss: 'did:web:claims.web3.storage',
  aud: 'did:web:claims.web3.storage',
  v: '0.9.1',
  s: {
    '/': {
      bytes: '7aEDQJOpYFb4BilDcbMMK0YuLnkeCqVpFadt/ZGnaVEkb6aiQ2KzXUYkyBxyGhixA6GePjSrf21hDXxd79Wck6gklQQ'
    }
  },
  exp: null,
  att: [
    {
      can: 'assert/equals',
      nb: {
        content: {
          '/': 'bafkzcibbai3tdo4zvruj6zxo6wlt4suu3imi6to4vzmaojh4n475mdp5jcbtg'
        },
        equals: {
          '/': 'bagbaieratdhefxxpkhkae2ovil2tcs7pfr2grvabvvoykful7k2maeepox3q'
        }
      },
      with: 'did:web:claims.web3.storage'
    }
  ],
  prf: [],
  '/': 'bafyreibb5x5sw2kw6kcr7pjrho5evli7e3n4uohc63ek3vh5dxvw42feeq'
}

❯ ./bin.js write equals-demo-2.claim 
(node:29766) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
{ ok: {} }
```